include/psa/crypto_values.h

```diff
@@ -538,6 +611,14 @@
                        ((type) & PSA_KEY_TYPE_DH_GROUP_MASK) :  \
                        0))
 
+/** Diffie-Hellman groups defined in RFC 7919 Appendix A.
+ *
+ * This family includes groups with the following key sizes (in bits):
+ * 2048, 3072, 4096, 6144, 8192. A given implementation may support
+ * all of these sizes or only a subet.
+ */
+#define PSA_DH_GROUP_RFC7919            ((psa_dh_group_t) 0x02)
```

Comment:
Presume this should bx `0x020000`? - to align with the definition of the ECC group families?

Reply:
At least - when first introduced, until the commit that changes to 16-bit key types...

Reply:
I amended the old commits.